charts/csi-baremetal-deployment/values.yaml

```diff
@@ -124,6 +124,7 @@ scheduler:
     image:
       tag:
 
+    url_prefix: http://127.0.0.1:8889
```

Comment:
I suggest to pass port number only. Don't think that we need to set IP address in values as well.

Reply:
Fixed. New parameter is called `extender_port`

Reply:
Please use:
```
scheduler:
  extender:
    port: 8889
```

Reply:
Fixed. It represents in deployment CRD as `extenderPort` to not create new data struct with one field